BICI_Psoc_workspace/ProximalBack_V3.cydsn/main.c

```diff
@@ -83,7 +83,7 @@ int main(void)
                     i2cReadBuffer[i+51] = CapSense_dsRam.snsList.row5[i].raw[0];  
                     i2cReadBuffer[i+57] = CapSense_dsRam.snsList.row6[i].raw[0];
                 }
-                for(unsigned int i=0; i<5; ++i) // Row7-10
+                for(unsigned int i=0; i<4; ++i) // Row7-10

```

Comment:
What is this change? Whatever it is, it will be rewriting over old values... Did you test your code?

Reply:
in rows 7-10 there are 5 taxels not 4

Reply:
Ok, but again "it will be rewriting over old values... Did you test your code?". 63+4 = 67, etc...

Reply:
Sorry, I'm not sure if I understand what you mean